[bugfix] install phantomjs (fixes #12)

mitsururike opened the pull request.

Commit:
[bugfix] install phantomjs (fixes #12)

trombik commented on README.md:
```diff
@@ -58,15 +58,29 @@ dependencies:
+  vars_files:
+    - ansible_vault_key.yml
   roles:
+    - reallyenglish.apt-repo
```
IIRC, if you add dependencies to meta, these can be omitted because `ansible-role-java` depends on `reallyenglish.apt-repo`. a drawback is, the depended roles are always executed before the depending role no matter how. the point is, when some roles are always required for successful play, the role should go to meta. if they are optional, and required for the purpose of demonstration in an example, they should go to `roles` in the playbook.

which is the case here?

trombik commented on tasks/install-Debian.yml:
```diff
@@ -19,3 +19,8 @@
     name: rake
     state: present
+
+- name: Install phantomjs
```
this. as well as `rake`, should have a comment why phantomjs is installed because not all jenkins users would not expect it to be installed.

trombik replied:
probably, it is better to make them optional, like

```yaml
- name: Install jenkins_extra_packages
  package: "{{ item }}"
    state: present
  with_items: "{{ jenkins_extra_packages }}"
``` 

and a similar task for npm packages.

Commit:
remove unnecessary dependency

reallyenglish.apt-repo is required by ansible-role-java and
not have to be here.

Commit:
[feature] introducce variables to install extra packages
